packages/flutter/test/material/range_slider_test.dart

```diff
@@ -1532,7 +1532,7 @@ void main() {
         );
       }
 
-      await tester.pumpWidget(buildApp(divisions: 3));
```

Comment:
The code change above makes sense to me, but why the test will fail when the divisions stays 3?